jenkins/databricks/build.sh

```diff
@@ -80,7 +80,7 @@ mvn install:install-file \
 mvn -Pdatabricks clean verify -DskipTests
 
 # copy so we pick up new built jar
-sudo cp dist/target/rapids-4-spark_2.12-*-SNAPSHOT.jar /databricks/jars/rapids-4-spark_2.12-0.2.0-SNAPSHOT-ci.jar
+sudo cp dist/target/rapids-4-spark_2.12-*-SNAPSHOT.jar /databricks/jars/rapids-4-spark_2.12-0.1.0-SNAPSHOT-ci.jar
```

Comment:
note this has to overwrite the file that is hardcoded on our databricks cluster that is currently statically deployed so it has to stay 0.1

Reply:
Can we add a comment explaining that?  Otherwise I'm sure someone is going to search-n-replace this eventually when they shouldn't.

Reply:
updated